.github/workflows/dists.yml

```diff
@@ -41,18 +41,22 @@ jobs:
         # pin to the latest supported python version
         python-version: "3.13"
         # x86 python needs to be available for the win32 wheel
-        architecture: ${{ ( matrix.os == 'windows-latest' && matrix.target == 'i686' ) && 'x86' || null }}
+        architecture: ${{ ( matrix.os == 'windows-2025' && matrix.target == 'i686' ) && 'x86' || null }}
+
+    - name: Add python lib to PATH
+      if: matrix.os == 'windows-2025'
+      run: Add-Content $env:GITHUB_PATH "${{ env.pythonLocation }}\lib"
 
     - uses: PyO3/maturin-action@v1
       with:
         target: ${{ matrix.target }}
         manylinux: ${{ matrix.manylinux || 'auto' }}
         # Keep in sync with tests.yml
-        args: --release --out dist --interpreter '3.8 3.9 3.10 3.11 3.12 3.13 3.13t'
+        args: --release --out dist --interpreter ${{ (matrix.os == 'windows-2025' && '3.8 3.9 3.10 3.11 3.12 3.13') || '3.8 3.9 3.10 3.11 3.12 3.13 3.13t' }}
```

Comment:
hi @davidhewitt :wave: 

3.13t no longer works, it was still working 2 weeks ago with all the same versions: https://github.com/oconnor663/blake3-py/pull/49

removing it here for windows gets us green again

Reply:
resolved with 86876d5e